### PR TITLE
Use flutter_driver test outputs dir in Gallery test

### PR DESCRIPTION
**Pull request by cbracken**

### CODE REVIEW

File: examples/flutter_gallery/test_driver/transitions_perf_test.dart [hunk 1]
```diff
@@ -136,14 +134,16 @@ void main() {
       // Save the duration (in microseconds) of the first timeline Frame event
       // that follows a 'Start Transition' event. The Gallery app adds a
       // 'Start Transition' event when a demo is launched (see GalleryItem).
+      await _fs.directory(testOutputsDirectory).create(recursive: true);
```
Comment (yjbanov) [inline examples/flutter_gallery/test_driver/transitions_perf_test.dart]: Should we move this into `writeSummaryToFile` and `writeTimelineToFile`?

**Review by yjbanov — APPROVED** [review]

File: examples/flutter_gallery/test_driver/transitions_perf_test.dart [hunk 2]
```diff
@@ -87,9 +87,7 @@ Future<Null> saveDurationsHistogram(List<Map<String, dynamic>> events) async {
   }
 
   // Save the durations Map to a file.
-  final String destinationDirectory = 'build';
-  await _fs.directory(destinationDirectory).create(recursive: true);
-  final File file = _fs.file(path.join(destinationDirectory, 'transition_durations.timeline.json'));
+  File file = await _fs.file(outputPath);
```
Comment (yjbanov) [inline examples/flutter_gallery/test_driver/transitions_perf_test.dart]: This seems to be synchronous API.

**Review by yjbanov — APPROVED** [review]

File: examples/flutter_gallery/test_driver/transitions_perf_test.dart (hunk 1 above)
Comment (cbracken) [inline examples/flutter_gallery/test_driver/transitions_perf_test.dart]: Those actually already do that. Moved the create call into the write method above to keep it all symmetric.

File: examples/flutter_gallery/test_driver/transitions_perf_test.dart (hunk 2 above)
Comment (cbracken) [inline examples/flutter_gallery/test_driver/transitions_perf_test.dart]: Meant to revert that! Fixed by above change.

**Pull request merged**